org.eclipse.m2e.binaryproject.ui/src/org/eclipse/m2e/binaryproject/ui/internal/BinaryProjectImportWizard.java

```diff
@@ -57,7 +57,7 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
       }
     }
     artifactsPage =
-        new MavenDependenciesWizardPage(new ProjectImportConfiguration(), "Artifacts", "Select artifacts to import") {
+        new MavenDependenciesWizardPage(new ProjectImportConfiguration(), "Artifacts", "Select artifacts to import") { //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
This should typically be internationalized; hence `//$NON-NLS-1$` is erroneous here.